webtau-core/src/main/java/com/twosigma/webtau/expectation/equality/GreaterThanMatcher.java

```diff
@@ -74,4 +74,9 @@ public boolean negativeMatches(ActualPath actualPath, Object actual) {
         compareToComparator = CompareToComparator.comparator();
         return compareToComparator.compareIsLessOrEqual(actualPath, actual, expected);
     }
+
+    @Override
+    public String toString() {
+        return GreaterLessEqualMatcherRenderer.render(this, compareToComparator, expected);
```

Comment:
Some static function imports for things like `render` and `comparator()` might actually make the code nicer but it's also a stylistic choice so up to you.

Reply:
when I static import outside of DSL I prefer method itself to be less ambiguous. Any suggestion for the class name itself?